functions-and-operators/tidb-functions.md

```diff
@@ -254,13 +172,9 @@ Check Table Before Drop: false
 1 row in set (0.00 sec)
 ```
 
-### MySQL compatibility
-
-The `TIDB_VERSION` function is TiDB-specific and not compatible with MySQL. If MySQL compatibility is required, you can also use `VERSION` to get version information, but the result does not contain build details.
```

Comment:
I would prefer to keep this info somehere. 

Another compatible way of doing this is via the `version%` variables:
```
 MySQL  127.0.0.1:4000 ssl  test  SQL > show variables like 'version%';
+-------------------------+---------------------------------------------------------------------------+
| Variable_name           | Value                                                                     |
+-------------------------+---------------------------------------------------------------------------+
| version                 | 5.7.25-TiDB-v5.3.0-alpha-228-g74b329402                                   |
| version_comment         | TiDB Server (Apache License 2.0) Enterprise Edition, MySQL 5.7 compatible |
| version_compile_machine | x86_64                                                                    |
| version_compile_os      | osx10.8                                                                   |
+-------------------------+---------------------------------------------------------------------------+
4 rows in set, 1 warning (0.0029 sec)
```

Reply:
I included a briefer version of this text in the table:

> | [`TIDB_VERSION()`](#tidb_version) | The `TIDB_VERSION()` function returns the TiDB version with additional build information. |

i.e. The title of this page is TiDB specific syntax. The previous text was too verbose talking about "not being MySQL compatible" (that part should be obvious).

Reply:
The thing I would keep in there is that people can use `VERSION()` instead of `TIDB_VERSION()` if they want compatibility. If you don't think that makes sense, feel free to let it out.